.github/workflows/ci.yml

```diff
@@ -88,10 +88,8 @@ jobs:
       platforms: linux/${{ matrix.platform }}
       dockerfile: build/devdeps.Dockerfile
       toolchain: ${{ matrix.toolchain }}
-      build_args: |
-        llvm_commit=${{ needs.metadata.outputs.llvm_commit }}
-        pybind11_commit=${{ needs.metadata.outputs.pybind11_commit }}
       registry_cache_from: ${{ inputs.cache_base || needs.metadata.outputs.cache_base }}
+      checkout_submodules: true
```

Comment:
Noting that, I think using `depth=1` with the llvm-project submodule may be an issue. At least I encountered problems with it. It may be because the commit we're stuck on is very old at this time.